linux_os/guide/system/bootloader-zipl/zipl_bls_entries_only/oval/shared.xml

```diff
@@ -0,0 +1,27 @@
+<def-group>
+  <definition class="compliance" id="zipl_bls_entries_only" version="1">
+    <metadata>
+      <title>Ensure zIPL entries are BLS compliant</title>
+      {{{- oval_affected(products) }}}
+      <description>Check if /etc/zipl.conf configures any boot entry</description>
+    </metadata>
+    <criteria operator="AND">
+      <criterion test_ref="test_zipl_bls_entries_only"
+      comment="Test presence of image configuration in /etc/zipl.conf" />
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test id="test_zipl_bls_entries_only"
+  comment="Test presence of image configuration in /etc/zipl.conf"
+  check="all" check_existence="none_exist" version="1">
+    <ind:object object_ref="object_zipl_bls_entries_only" />
+  </ind:textfilecontent54_test>
+
+  <ind:textfilecontent54_object id="object_zipl_bls_entries_only"
+  version="1">
+    <ind:filepath operation="pattern match">^/etc/zipl.conf$</ind:filepath>
+    <ind:pattern operation="pattern match">^image\s*=.*$</ind:pattern>
```

Comment:
I found this example: https://insights-core.readthedocs.io/en/latest/shared_parsers_catalog/zipl_conf.html It indicates that there may be leading whitespaces before `image`.

Reply:
Great finding, the check has been improved.